### DISCUSSION TITLE
Tasmota doesn't register an external switchwire toggle on my Sonoff Mini R4

### DISCUSSION THREAD

**Discussion by GijsTimmers** [arendst/Tasmota#20693] [Q&A]
I am running the [Blakadder template](https://templates.blakadder.com/sonoff_MINIR4.html) on a Sonoff Mini R4. My goal is to toggle the bathroom fan whenever the light is switched on. There's a switch wire connected to the wall switch which is going into S2; S1 remains unused. This should be possible, [according to the manual](https://sonoff.tech/wp-content/uploads/2023/05/%E8%AF%B4%E6%98%8E%E4%B9%A6-MINIR4-V1.1.pdf).

However, when toggling the switch, nothing happens. I don't see anything pop up in the console either. 

I am running Tasmota 13.3.0(tasmota32). 

Help would be appreciated.

Current setup:
![2](https://github.com/arendst/Tasmota/assets/972314/b439516b-86ac-4c42-a048-0b61ba8a23d3)

<img src="https://github.com/arendst/Tasmota/assets/972314/38e53bd8-6069-45ba-8ffd-1a42f25f34bb" width="300" height="auto">

**Comment by Noschvie** [comment]
Did you check the console logs? Habe a look to switch 1 and use rules.

**Reply by barbudor** [reply]
The diagram shows 5 wires connected to the R4
The photo shows 3

Don't you think you are missing something?

**Reply by GijsTimmers** [reply]
As I wrote, nothing pops up in the console log.

**Reply by GijsTimmers** [reply]
That's right, L out and N (out) are not connected yet, but it shouldn't matter to the switch behaviour or relay state.

**Reply by barbudor** [reply]
What happens if you use the default cabling ?
![image](https://github.com/arendst/Tasmota/assets/2996042/d0629151-fa23-476f-a34c-b6224c4161ea)

**Reply by GijsTimmers** [reply]
It will probably work right but that's not my goal. There's a switch wire running in my bathroom and I want to use it to control the fan.

**Reply by barbudor** [reply]
If you don't care about suggestions to help you trouble shoot, I can't do anything more
bye

**Comment by GijsTimmers** [comment]
> What happens if you use the default cabling ? ![image](https://private-user-images.githubusercontent.com/2996042/303947405-d0629151-fa23-476f-a34c-b6224c4161ea.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc3NjAwNTYsIm5iZiI6MTcwNzc1OTc1NiwicGF0aCI6Ii8yOTk2MDQyLzMwMzk0NzQwNS1kMDYyOTE1MS1mYTIzLTQ3NmYtYTM0Yy1iNjIyNGM0MTYxZWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMTJUMTc0MjM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NmE5NzdmM2MzYThjYTNmZjgwZmEzOGU1ZWNhZDdkYmU2YzE4ZjRjOTI0NzNhZDZlM2U5MDNmNDYxMTk5ZjYyNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.yFVTB3QgnTjn9fBuaU_wvnkbWOWaqVuEC_DqXzEkquc)

When using this cabling, the switch works normally.

![20240212_183811](https://github.com/arendst/Tasmota/assets/972314/cfe73b4e-8437-428f-920c-9dd1d762936f)

**Reply by barbudor** [reply]
That means the hardware is OK and the template seems OK too

Do you have one miniR4 spare with the original firmware to confirm that the alternate cabling mode works with the original firmware? 

Without the device in hand, it's hard to guess what could be missing.

**Reply by GijsTimmers** [reply]
I set up the device in a testing setup now and when doing this, the device works normally. So I guess it must be a cable issue then. 

![20240212_185151](https://github.com/arendst/Tasmota/assets/972314/3a872577-d202-4cde-a572-5cc216cb0259)

**Comment by GijsTimmers** [comment]
Marking as closed as it's probably a wiring issue or hardware issue.
